### PR TITLE
V. Petrat - Added Google search test.

### PR DESCRIPTION
**Pull request by VladimirPetrat**
Added multiple browser support. Errors in names of fields and methods are fixed.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -33,7 +33,13 @@
             <groupId>org.seleniumhq.selenium</groupId>
             <artifactId>selenium-chrome-driver</artifactId>
             <scope>test</scope>
-            <version>${chrome.driver.version}</version>
+            <version>${selenium.java.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-support</artifactId>
```
Comment (toleksyn) [inline pom.xml]: why is this needed?

File: src/main/java/com/softserveinc/ita/vPetrat/pageObjects/GoogleHomePage.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+public class GoogleHomePage {
+
+    private  By searchInput = By.xpath("//input[@class='gLFyf gsfi']");
+
+    public GoogleHomePage openHomePage() {
+        TestRunner.getDriver().get("https://www.google.com/");
+        return this;
+    }
+
+    public GoogleSearchResultPage searchForTheSpecifiedRequest(String request) {
```
Comment (toleksyn) [inline src/main/java/com/softserveinc/ita/vPetrat/pageObjects/GoogleHomePage.java]: rename

File: src/main/java/com/softserveinc/ita/vPetrat/pageObjects/GoogleHomePage.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        TestRunner.getWait().until(ExpectedConditions.elementToBeClickable(searchInput));
+        WebElement searchInputElement = TestRunner.getDriver().findElement(searchInput);
```
Comment (toleksyn) [inline src/main/java/com/softserveinc/ita/vPetrat/pageObjects/GoogleHomePage.java]: duplication

File: src/main/java/com/softserveinc/ita/vPetrat/resources/DriverType.java [hunk 4]
```diff
@@ -0,0 +1,57 @@
+package com.softserveinc.ita.vPetrat.resources;
+
+
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.opera.OperaDriver;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+public enum DriverType {
```
Comment (toleksyn) [inline src/main/java/com/softserveinc/ita/vPetrat/resources/DriverType.java]: wasn't part of task, didn't review

File: src/main/java/com/softserveinc/ita/vPetrat/resources/googleSearchRequest.properties [hunk 5]
```diff
@@ -0,0 +1 @@
+searchRequestKey=Funny Kitties
```
Comment (toleksyn) [inline src/main/java/com/softserveinc/ita/vPetrat/resources/googleSearchRequest.properties]: overcomplication

**Review by toleksyn — CHANGES_REQUESTED** [review]

**Comment by VladimirPetrat** [comment]
Comment taken into account. Test multiple result links added.

File: pom.xml [hunk 6]
```diff
@@ -33,7 +33,7 @@
             <groupId>org.seleniumhq.selenium</groupId>
             <artifactId>selenium-chrome-driver</artifactId>
             <scope>test</scope>
-            <version>${chrome.driver.version}</version>
+            <version>${selenium.java.version}</version>
```
Comment (toleksyn) [inline pom.xml]: do not change variables here, change to needed version in chrome.driver.version

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: src/test/java/com/softserveinc/ita/pageobjects_task/GoogleSearchTest.java [hunk 7]
```diff
@@ -0,0 +1,36 @@
+package com.softserveinc.ita.pageobjects_task;
```
Comment (toleksyn) [inline src/test/java/com/softserveinc/ita/pageobjects_task/GoogleSearchTest.java]: create your package - vpetrat - and move test there

File: src/main/java/com/softserveinc/ita/vPetrat/pageObjects/GoogleHomePage.java [hunk 8]
```diff
@@ -0,0 +1,22 @@
+package com.softserveinc.ita.vPetrat.pageObjects;
```
Comment (toleksyn) [inline src/main/java/com/softserveinc/ita/vPetrat/pageObjects/GoogleHomePage.java]: code convention, rename - vpetrat, pageobjects
package names are all lower case with underscores if needed

**Review by toleksyn — CHANGES_REQUESTED** [review]

**Pull request merged**